src/noit_filters.c

```diff
@@ -255,6 +257,20 @@ noit_filter_exists(const char *name) {
   UNLOCKFS();
   return exists;
 }
+
+int
+  noit_filter_get_seq(const char *name, int64_t *seq) {
+  int exists;
+  void *v;
+  LOCKFS();
+  exists = mtev_hash_retrieve(filtersets, name, strlen(name), &v);
+  UNLOCKFS();
+  if(exists) {
```

Comment:
Change to if(exists && seq != NULL) to better match convention in the rest of mtev/noit.

If the pass NULL in it will not crash and just return exists correctly.